tests/models/whisper/test_modeling_whisper.py

```diff
@@ -998,20 +1006,13 @@ class WhisperModelIntegrationTests(unittest.TestCase):
     def default_processor(self):
         return WhisperProcessor.from_pretrained("openai/whisper-base")
 
-    def _load_datasamples(self, num_samples):
-        ds = load_dataset("hf-internal-testing/librispeech_asr_dummy", "clean", split="validation")
-        # automatic decoding with librispeech
-        speech_samples = ds.sort("id").select(range(num_samples))[:num_samples]["audio"]
-
-        return [x["array"] for x in speech_samples]
-
     @slow
     def test_tiny_logits_librispeech(self):
         torch_device = "cpu"
         set_seed(0)
         model = WhisperModel.from_pretrained("openai/whisper-tiny")
         model.to(torch_device)
-        input_speech = self._load_datasamples(1)
+        input_speech = load_datasamples(1)
```

Comment:
I can understand putting this into a free function but if it's only used in one class, we generally keep it as a member function.

Reply:
for sure, removed this change. i'd been moving the tests around and at one point I had two classes using this